subsys/bluetooth/controller/ll_sw/ull_llcp.h

```diff
@@ -146,3 +146,13 @@ const uint8_t *ull_cp_chan_map_update_pending(struct ll_conn *conn);
  */
 uint8_t ull_cp_data_length_update(struct ll_conn *conn, uint16_t max_tx_octets, uint16_t max_tx_time);
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH */
+/**
```

Comment:
In case of declarations in header files conditional compilation is not needed. If the function is not used compiler will not complain about it. This is a common way for function declarations in header files in controller code.